meilix-artwork/debian/plymouth-meilix-logo.prerm

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh -e
+# This script can be called in the following ways:
+#
+# After the package was removed:
+#       <prerm> remove
+#
+# After the package was purged:
+#       <prerm> purge
+#
+# After the package was upgraded:
+#       <old-prerm> upgrade <new-version>
+# if that fails:
+#       <new-prerm> failed-upgrade <old-version>
+#
+#
+# After all of the packages files have been replaced:
+#       <prerm> disappear <overwriting-package> <version>
+#
+#
+# If preinst fails during install:
+#       <new-prerm> abort-install
+#
+# If preinst fails during upgrade of removed package:
+#       <new-prerm> abort-install <old-version>
+#
+# If preinst fails during upgrade:
+#       <new-prerm> abort-upgrade <old-version>
+
+case "$1" in
+    remove)
+        update-alternatives \
+            --remove default.plymouth /usr/share/plymouth/themes/meilix-logo/meilix-logo.plymmouth
```

Comment:
Are you sure it is to be spelled plymmouth?

Reply:
Sorry @tabesin, It should be plymouth, I will add it in my latest commit

Reply:
@tabesin  Please review now